Update to Ultimate College of Winterhold

ddbb07 opened the pull request:
Made patches from "Ultimate College of Winterhold" recognized as patches for "Obscure's College of Winterhold"

ddbb07 committed:
Update to "OCW_Obscure's_CollegeofWinterhold.esp"

Made patches from "Ultimate College of Winterhold" recognized as patches for "Obscure's College of Winterhold"

MacSplody commented on masterlist.yaml:
```diff
@@ -15004,7 +15004,7 @@ plugins:
       - <<: *patchIncluded
         subs: [ 'Enhanced Lights and FX' ]
-        condition: 'active("EnhancedLightsandFX.esp") and not active("OCW_ELFX_FEPatch.esp")'
+        condition: 'active("EnhancedLightsandFX.esp") and not active("OCW_ELFX_FEPatch.esp") and not active("UltimateCollege_ELFX_Patch.esp")'
```
It would probably be best to always hide these messages if `UltimateCollege.esp`is installed.
```suggestion
        condition: 'active("EnhancedLightsandFX.esp") and not active("OCW_ELFX_FEPatch.esp") and not active("UltimateCollege.esp")'
```

ddbb07 replied:
I'll change it to that

ddbb07 committed:
Update to "OCW_Obscure's_CollegeofWinterhold.esp"

Update so that patches from "Ultimate College of Winterhold" are recognized as patches for "Obscure's College of Winterhold"

ddbb07 commented:
I'll close this until im done

MacSplody commented:
> 
> 
> I'll close this until im done

Okay, you can mark it as a draft iirc, if it's unfinished.
You don't have to get it all done in one commit either, the commits can be squashed on merge if needed.

ddbb07 committed:
Update to Ultimate College of Winterhold

Added patch messages for "UltimateCollege.esp"

ddbb07 commented:
How do i mark it as a draft
Edit: I found it

ddbb07 committed:
Update to Ultimate College of Winterhold

Made patches from "Ultimate College of Winterhold" recognized as patches for "Obscure's College of Winterhold"

ddbb07 committed:
Update to Ultimate College of Winterhold

Made patches from "Ultimate College of Winterhold" recognized as patches for "Immersive College of Winterhold"

ddbb07 commented:
Why is it failing the check

sibir-ine commented on masterlist.yaml:
```diff
@@ -15004,10 +15004,10 @@ plugins:
       - <<: *patchIncluded
         subs: [ 'Enhanced Lights and FX Exteriors' ]
-        condition: 'active("ELFX - Exteriors.esp") and not active("OCW_ELFXE_FEPatch.esp")'
+        condition: 'active("ELFX - Exteriors.esp") and not active("OCW_ELFXE_FEPatch.esp")' and not active("UltimateCollege_ELFXExt_Patch.esp") and not active("UltimateCollege.esp")'
```
```suggestion
        condition: 'active("ELFX - Exteriors.esp") and not active("OCW_ELFXE_FEPatch.esp") and not active("UltimateCollege_ELFXExt_Patch.esp") and not active("UltimateCollege.esp")'
```

ddbb07 replied:
Thanks

ddbb07 committed:
Fix

Co-authored-by: sibir-ine <69141501+sibir-ine@users.noreply.github.com>

ddbb07 commented:
Im finished now

MacSplody commented on masterlist.yaml:
```diff
@@ -13560,16 +13560,16 @@ plugins:
       - *patch3rdPartySRPC
       - <<: *patch3rdParty_LotD
-        condition: 'active("LegacyoftheDragonborn.esm") and not active("DBM_ImmersiveCollegeofWinterhold_Patch.esp") and (version("LegacyoftheDragonborn.esm", "5.0.24", >=) or (not checksum("LegacyoftheDragonborn.esm", 31563183) and not checksum("LegacyoftheDragonborn.esm", 6736FB94) and not checksum("LegacyoftheDragonborn.esm", CB82480D)))'
+        condition: 'active("LegacyoftheDragonborn.esm") and not active("DBM_ImmersiveCollegeofWinterhold_Patch.esp") and not active("UltimateCollege_LOTD_Patch.esp") and not active("UltimateCollege.esp") and (version("LegacyoftheDragonborn.esm", "5.0.24", >=) or (not checksum("LegacyoftheDragonborn.esm", 31563183) and not checksum("LegacyoftheDragonborn.esm", 6736FB94) and not checksum("Legacyoft ...
```
> and not active("UltimateCollege_LOTD_Patch.esp") and not active("UltimateCollege.esp")

Checking for `UltimateCollege.esp` should be enough to hide the message.
It's not going to cause any harm to check for the patch as well, it just seems redundant.
Is there some other reason that I'm missing?

```suggestion
        condition: 'active("LegacyoftheDragonborn.esm") and not active("DBM_ImmersiveCollegeofWinterhold_Patch.esp") and not active("UltimateCollege.esp") and (version("LegacyoftheDragonborn.esm", "5.0.24", >=) or (not checksum("LegacyoftheDragonborn.esm", 31563183) and not checksum("LegacyoftheDragonborn.esm", 6736FB94) and not checksum("LegacyoftheDragonborn.esm", CB82480D)))'
```

ddbb07 replied:
No theres not, i'll remove it.

ddbb07 committed:
Removed check for UCOW patches

MacSplody commented:
@ddbb07 Thanks.

themayor897 commented:
Thanks!
